pkg/recovery/securerecoverer.go

```diff
@@ -0,0 +1,53 @@
+package recovery
+
+import (
+	"log"
+	"os"
+	"syscall"
+	"time"
+)
+
+const debugTimeout time.Duration = 10
```

Comment:
make this a public variable so it can be overridden. Also useful for unit testing

Reply:
Done